linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_data_retention_max_log_file_action/rule.yml

```diff
@@ -55,12 +55,18 @@ references:
     srg: SRG-OS-000047-GPOS-00023
     stigid@ol8: OL08-00-030050
 
-ocil_clause: 'the system has not been properly configured to rotate audit logs'
+ocil_clause: 'the system has not been properly configured to take appropriate action'
 
 ocil: |-
     Inspect <tt>/etc/audit/auditd.conf</tt> and locate the following line to
-    determine if the system is configured to rotate logs when they reach their
+    determine if the system is configured to take appropriate action when they reach their
     maximum size:
     <tt>$ sudo grep max_log_file_action /etc/audit/auditd.conf</tt>
-    <pre>max_log_file_action <tt>rotate</tt></pre>
+    <pre>max_log_file_action = {{{ xccdf_value("var_auditd_max_log_file_action") }}}</pre>
```

Comment:
Would other values be allowed, similar to other rules in this PR?

Reply:
the Oracle Linux 8 STIG has the following:

```
Details
Check Text (C-52161r779745_chk)

Verify that the SA and ISSO (at a minimum) are notified when the audit storage volume is full. 
 
Check which action OL 8 takes when the audit storage volume is full with the following command: 
 
$ sudo grep max_log_file_action /etc/audit/auditd.conf 
 
max_log_file_action=syslog 
 
If the value of the "max_log_file_action" option is set to "ignore", "rotate", or "suspend", or the line is commented out, this is a finding.
Fix Text (F-52115r779746_fix)

Configure OL 8 to notify the SA and ISSO when the audit storage volume is full by configuring the "max_log_file_action" parameter in the "/etc/audit/auditd.conf" file with the a value of "syslog" or "keep_logs": 
 
max_log_file_action=syslog
```

it prohibits the usage of "ignore", "rotate", or "suspend"

and allows  "syslog" or "keep_logs"

maybe we can use it as inspiration

Reply:
I have add this  text here.

So now we have a different problem, in all rules in this PR:

The OCIL and fixtext isn't aligned with OVAL and Bash, because the OCIL and fixtext allow multiple values, but OVAL and bash uses specifically only the value set as variable in the SRG/control file.